heartbeat/sender/multikeyHeartbeatSender.go

```diff
@@ -186,7 +186,7 @@ func (sender *multikeyHeartbeatSender) sendMessageForKey(pkBytes []byte) error {
 	}
 
 	sender.mainMessenger.BroadcastUsingPrivateKey(sender.topic, buff, pid, p2pSk)
-	sender.fullArchiveMessenger.BroadcastUsingPrivateKey(sender.topic, buff, pid, p2pSk) // TODO[Sorin]: rethink if we need to send this
+	sender.fullArchiveMessenger.BroadcastUsingPrivateKey(sender.topic, buff, pid, p2pSk)
```

Comment:
I do not think we need this. This network should not care about virtual peers as its purpose is for data request/response

Reply:
removed